tool.ts

```diff
@@ -22,12 +22,15 @@ export function debug(message: string): void {
 
 export function prependPath(toolPath: string) {
     tl.assertAgent('2.115.0');
-    tl.debug('prepend path: ' + toolPath);
-    if (!tl.exist(toolPath)) {
-        throw new Error('Path does not exist: ' + toolPath);
+    if (!toolPath) {
+        throw new Error('Parameter toolPath must not be null or empty');
+    }
+    else if (!tl.exist(toolPath) || !tl.stats(toolPath).isDirectory()) {
+        throw new Error('Directory does not exist: ' + toolPath);
     }
 
     // todo: add a test for path
+    console.log('Prepending PATH environment variable with directory:' + toolPath);
```

Comment:
Prepending the path is significant, therefore should always console.log (we can add a silent option later if required).